### PR TITLE
Signal-Desktop: update to 7.10.0

### PR DESCRIPTION
**Pull request by MIvanchev**
#### Testing the changes
- I tested the changes in this PR: **NO**

### CODE REVIEW

**Comment by MIvanchev** [comment]
Signal is currently unusable because the current version is too old, the program displays an error message.

**Comment by MIvanchev** [comment]
WARNING: A small patching changes the required NodeJS version from latest to what we currently have (18.19.1). I have no idea about the security implications.

**Comment by okawo80085** [comment]
Closes #50481

File: srcpkgs/Signal-Desktop/template [hunk 1]
```diff
@@ -14,14 +14,16 @@ maintainer="Orphaned <orphan@voidlinux.org>"
...
 nostrip_files="signal-desktop"
 
 post_extract() {
 	# git-lfs hook needs to be installed for one of the dependencies
 	git lfs install
 
-	vsed 's/"node": "/&>=/' -i package.json
+	# FIXME: We currently don't have the latest Node so let's force Signal
+	# to use what we have. This is very far from ideal.
+	vsed 's/"node": ".*"/"node": ">=18.19.1"/' -i package.json
```
Comment (ahesford) [inline srcpkgs/Signal-Desktop/template:26]: You should use a real patch for this, not `vsed`.

**Review by ahesford — CHANGES_REQUESTED** [review]

File: srcpkgs/Signal-Desktop/template (hunk 1 above)
Comment (MIvanchev) [reply srcpkgs/Signal-Desktop/template:26]: I thought so too but decided to stick to what was already there :/ Let's address this next cycle... I just noticed it was maybe copied from Arch.

Comment (leahneukirchen) [reply srcpkgs/Signal-Desktop/template:26]: I will take a stab on Node 20 on the weekend so this won't be for too long I hope.

**Pull request merged**